src/applications/personalization/dashboard-2/tests/e2e/Dashboard.cypress.spec.js

```diff
@@ -0,0 +1,61 @@
+import disableFTUXModals from '~/platform/user/tests/disableFTUXModals';
+import { MY_VA_ROOT } from 'applications/personalization/dashboard-2/constants';
```

Comment:
could we use the manifest to get the root url? https://github.com/department-of-veterans-affairs/vets-website/blob/c4b97f33c7c18b93d4aaa915109f8cc23cdb045a/src/applications/personalization/dashboard/manifest.json#L4-L5

Reply:
We sure can, updated! 👍